test/e2e/clusterctl_upgrade.go

```diff
@@ -390,7 +389,7 @@ func ClusterctlUpgradeSpec(ctx context.Context, inputGetter func() ClusterctlUpg
 		By("Waiting for the machines to exist")
 		Eventually(func() (int64, error) {
 			var n int64
-			machineList := &clusterv1alpha3.MachineList{}
+			machineList := &clusterv1.MachineList{}
```

Comment:
I would guess this has to be v1alpha4 for the v1alpha4 clusterctl upgrade test

IIRC depending on the test case v1beta1 doesn't exist in the CRD at this point (should only affect the v1alpha4 test now)

(similar below)